baremaps-config/pom.xml

```diff
@@ -14,6 +14,10 @@
       <groupId>com.baremaps</groupId>
       <artifactId>baremaps-blob</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Can't we simply get rid of the baremaps-config module completely?

Reply:
The remaining stuff is still needed, we could move it elsewhere though.

Reply:
Yes, we could get rid of the mapping package completly. The remaining classes (BlobMapper, BlobMapperException and VariableUtils) may be moved to the baremaps-blob package. What do you think?

Reply:
Done in https://github.com/baremaps/baremaps/pull/338/commits/83c9a9ae03641354aaa9ec7c83dbfd2dd6e9ecef